tests/unit/timing/strategies/test_fixed_schedule.py

```
@@ -227,6 +227,33 @@ async def test_non_final_turn_schedules_next(self) -> None:
         await strategy.handle_credit_return(credit)
         scheduler.schedule_at_perf_sec.assert_called_once()
 
+    async def test_continuation_turn_carries_has_forks(self) -> None:
+        """has_forks from the NEXT turn's metadata must ride onto the
+        continuation turn (the sticky router defers parent-entry eviction until
+        DAG children drain). Regression: from_previous_credit(credit) was called
+        without next_meta, so every continuation turn got has_forks=False."""
+        strategy, scheduler, _ = make_strategy([(0, "c1"), (100, "c1")])
```

Ruff (0.15.20)

[warning] Unpacked variable `scheduler` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)